Improved social icons

tanya-gupta25 opened the pull request:
- Instead of images of social icons , I've added actual icons from font awesome and added some styling & color to it.
![image](https://user-images.githubusercontent.com/82203261/185968300-7f170914-5294-46fa-8060-86aab6b561ec.png)

- Added hover effect on each icons & added target as blank so they open in a new page & also added links
- Instagram
![image](https://user-images.githubusercontent.com/82203261/185968383-66dc8f39-def8-4c41-9144-ca626b8d0f33.png)

- Twitter
![image](https://user-images.githubusercontent.com/82203261/185968467-4a368468-77ab-403f-b728-2b42af3981a6.png)

- Facebook
![image](https://user-images.githubusercontent.com/82203261/185968495-5420767b-f115-4945-b213-5143648c4d05.png)

TejalKhade28 commented:
> * Instead of images of social icons , I've added actual icons from font awesome and added some styling & color to it.
>   ![image](https://user-images.githubusercontent.com/82203261/185968300-7f170914-5294-46fa-8060-86aab6b561ec.png)
> * Added hover effect on each icons & added target as blank so they open in a new page & also added links
> * Instagram
>   ![image](https://user-images.githubusercontent.com/82203261/185968383-66dc8f39-def8-4c41-9144-ca626b8d0f33.png)
> * Twitter
>   ![image](https://user-images.githubusercontent.com/82203261/185968467-4a368468-77ab-403f-b728-2b42af3981a6.png)
> * Facebook
>   ![image](https://user-images.githubusercontent.com/82203261/185968495-5420767b-f115-4945-b213-5143648c4d05.png)

Which links did you give ?

tanya-gupta25 commented:
> > * Instead of images of social icons , I've added actual icons from font awesome and added some styling & color to it.
> >   ![image](https://user-images.githubusercontent.com/82203261/185968300-7f170914-5294-46fa-8060-86aab6b561ec.png)
> > * Added hover effect on each icons & added target as blank so they open in a new page & also added links
> > * Instagram
> >   ![image](https://user-images.githubusercontent.com/82203261/185968383-66dc8f39-def8-4c41-9144-ca626b8d0f33.png)
> > * Twitter
> >   ![image](https://user-images.githubusercontent.com/82203261/185968467-4a368468-77ab-403f-b728-2b42af3981a6.png)
> > * Facebook
> >   ![image](https://user-images.githubusercontent.com/82203261/185968495-5420767b-f115-4945-b213-5143648c4d05.png)
> 
> Which links did you give ?

Hackclub RAIT links which were mentioned earlier

TejalKhade28 commented:
> > > * Instead of images of social icons , I've added actual icons from font awesome and added some styling & color to it.
> > >   ![image](https://user-images.githubusercontent.com/82203261/185968300-7f170914-5294-46fa-8060-86aab6b561ec.png)
> > > * Added hover effect on each icons & added target as blank so they open in a new page & also added links
> > > * Instagram
> > >   ![image](https://user-images.githubusercontent.com/82203261/185968383-66dc8f39-def8-4c41-9144-ca626b8d0f33.png)
> > > * Twitter
> > >   ![image](https://user-images.githubusercontent.com/82203261/185968467-4a368468-77ab-403f-b728-2b42af3981a6.png)
> > > * Facebook
> > >   ![image](https://user-images.githubusercontent.com/82203261/185968495-5420767b-f115-4945-b213-5143648c4d05.png)
> > 
> > 
> > Which links did you give ?
> > RAIT acm links which were mentioned earlier

Okay
